tests/Unit/Models/CentreModelTest.php

```diff
@@ -47,9 +47,15 @@ public function itCanHaveRegistrations()
     public function itCanHaveUsers()
     {
         $centre = factory(Centre::class)->create();
-        $centreUsers = factory(CentreUser::class, 3)->create([
-            'centre_id' => $centre->id,
-        ]);
+
+        factory(CentreUser::class, 3)
+            ->create()
+            ->each(
+                function (CentreUser $centreUser) use ($centre) {
+                    $centreUser->centres()->attach($centre);
```

Comment:
Does one of them need to be a homeCentre for this?

Reply:
Technically, yes, however, for the purposes of this test, that we can assign users, we can get away without it.
Add if you feel we should test that too.